src/components/importwms/ImportWmsDirective.js

```diff
@@ -186,6 +188,17 @@ goog.require('ga_urlutils_service');
               layer.Abstract = 'layer_invalid_no_name';
             }
 
+            // if there is no intersection between the map-extent
+            // and the WMS-layer-extent, the layer is set as invalid
+            layer.extent = getLayerExtentFromGetCap(layer);
+            layer.extent = ol.extent.intersects(
+                map.getView().getProjection().getExtent(), layer.extent);
+            if (!layer.extent) {
```

Comment:
if `ìntersects`returns a boolean , don't assign it to `layer.extent` propertyand use it diretly in the if. `layer.extent` must be an array of coordinates.